### ISSUE TITLE
Paintjob-dependent wheels are displayed incorrectly ingame

### ISSUE THREAD

**Issue by totixtreme** [0x1F9F1/Open1560#116]
The wheels are displayed incorrectly when they belong to different paintjobs. They always render in the primary paintjob, even though they render correctly in the vehicle selection screen. Here is an example, the blue wheels belong to the primary paintjob, and the red wheels belong to a different paintjob.

![SHOT0005](https://user-images.githubusercontent.com/112048596/186565890-84e53a11-2791-4ffc-978d-a0501db6d632.PNG)

![SHOT0006](https://user-images.githubusercontent.com/112048596/186565917-71c1e2d6-eb36-470d-bcc8-c38d9fed005a.PNG)

![SHOT0007](https://user-images.githubusercontent.com/112048596/186565925-88e0c62f-c85b-4c7c-a966-0d0a8db9a2e5.PNG)

![SHOT0008](https://user-images.githubusercontent.com/112048596/186565945-fbdfc73a-1be8-4877-a55d-a278ae501f1c.PNG)

**Comment by 0x1F9F1** [comment]
Yeah afaik only the BODY and FNDR[0/1] meshes are affected by the paintjob. The selection screen is different because it just uses the `H` mesh (which has the wheels and other parts built in).
